Dictionary.js

```diff
@@ -0,0 +1,81 @@
+<<<<<<< HEAD
```

Comment:
This text here means that there was a merge conflict, on need to pick content between <<<<<HEAD and ====== or between >>>>>>> and =========. You should get an option to see which content you want to keep.